src/components/vistas/scaner/index.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styles from './styles.module.scss';
+import { Link } from "react-router-dom";
+import scaner from '../../../assets/scaner.png';
+import Topbar from '../../reusable-components/topbar';
+
+function Scaner() {
+    return (
+        <div className={styles.Scaner}>
```

Comment:
Ten cuidado con que los nombres de las clases sean consistentes en el casing (mayúsculas/minúsculas).
```suggestion
        <div className={styles.scaner}>
```